### ISSUE TITLE
On MacOS 10.15.3, the wave appears but the spectrum does not

### ISSUE THREAD

**Issue by FuzzySockets** [markjay4k/Audio-Spectrum-Analyzer-in-Python#11]
On MacOS 10.15.3, the wave appears but the spectrum does not.

**Comment by hoppi13** [comment]
Was this with the PyQt version and if so did you ever figure it out? I am having the same issue with windows.

**Comment by FuzzySockets** [comment]
Yes, this was the PyQtGraph iteration and I never ended up solving the bug.

**Comment by hoppi13** [comment]
https://github.com/sbarratt/spectrum-analyzer/blob/master/sa.py 

I used this I hope it helps you too.
